app/components/common/eventCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+interface EventCardProps {
+  date: number;
+  month: string;
+  time: any;
+  eventTitle: string;
+  participantCount: any;
+}
+
+export default function EventCard({
+  date,
+  month,
+  time,
+  eventTitle,
+  participantCount,
+}: EventCardProps) {
+  return (
+    <>
+      <div className="border-2 w-64 h-72 bg-gray-200 rounded-3xl m-auto relative">
+        <div className="w-10 h-12 left-52 top-4 bg-white rounded-lg flex items-center flex-col relative divide-y">
+          <div className="text-black font-medium text-base">{date}</div>
```

Comment:
Should we validate, whether we are receiving proper date format here?

Reply:
So do we have to validate for `dd-mm-yyyy` and other date formats? and decided any one to be standard and expected for event card?

Reply:
Please look into this @harshith-venkatesh  : https://github.com/Real-Dev-Squad/website-calendar/pull/115#discussion_r1139247630